### DISCUSSION TITLE
discussion기능을 사용하기 이전에 지켜야할 룰이 있을까요?

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#1] [Q&A] [answered]
현재 라벨에 분리되어 있는 것 처럼 사용하면 될까요?

아니면 한가지 주제에 대해서 쓰레드로 토론형식으로 진행이 될까요..?

**Comment by jongfeel** [comment, marked as answer]
써보면서 분류해 보면 좋을 것 같습니다.

Q&A category가 기본으로 제공되니까 활용해 보면 좋을 것 같아요

**Reply by jongfeel** [reply]
discussions 글 하나에 질문을 한 개만 주제로 해서 했으면 좋겠고
쓰레드 토론 형식이 좋을 것 같습니다.

**Reply by fkdl0048** [reply]
넵 형식은 말씀하신 대로 맞춰서 작성하겠습니다..!  

혹시 멘토님을 태크하는 방법은 없나요?

**Reply by jongfeel** [reply]
제가 이미 초대가 되어 있기 때문에 메일이 옵니다.
업무 시간 중에 메일 확인은 거의 바로 하는편이에요
